src/borg/helpers.py

```diff
@@ -1879,15 +1888,10 @@ def hash_item(self, hash_function, item):
         return hash.hexdigest()
 
     def format_time(self, key, item):
-        t = self.time(key, item)
-        return format_time(t)
-
-    def format_time_json(self, key, item):
-        t = self.time(key, item)
-        return isoformat_time(t)
+        return OutputTimestamp(safe_timestamp(item.get(key) or item.mtime))
 
-    def time(self, key, item):
-        return safe_timestamp(item.get(key) or item.mtime)
+    def format_iso_time(self, key, item):
+        return self.format_time(key, item).to_json()
```

Comment:
strange: format_time is returning a formatter instance while format_iso_time returns a string.